src/renderer/src/personal-views/activity-view/activity-asset.ts

```diff
@@ -97,7 +98,7 @@ export class ActivityAsset extends LitElement {
       case 'pending':
         return html`<sl-skeleton style="height: 14px; width: 14px;" effect="pulse"></sl-skeleton> `;
       case 'complete':
-        return this.renderLogo(this.appletLogo.value.value);
+        return html`${this.renderLogo(this.appletLogo.value.value)}`;
```

Comment:
I think this change is obsolete, no? `this.renderLogo()` already returns an html template string...